tests/core/test_errors.py

```diff
@@ -324,3 +324,14 @@ def test_pickling_parser_error():
         assert unpickled.failure_cases == str(exc.failure_cases)
     else:
         pytest.fail("ParserError not raised")
+
+
+def test_unhashable_types_rendered_on_failing_checks_with_lazy_validation():
+    """Regression test for #1013, make sure the SchemaErrors are well-built."""
```

Comment:
let's expand `#1013` into a description of the issue so that maintainer doesn't have to look up a github url